pkg/sfu/downtrack.go

```diff
@@ -103,7 +106,9 @@ type DownTrack struct {
 
 	currentSpatialLayer atomicInt32
 	targetSpatialLayer  atomicInt32
-	temporalLayer       atomicInt32
+
+	currentTemporalLayer atomicInt32
```

Comment:
making this simpler, it was using a single variable and packing 16 bits of current and 16 bits of target. Just make it two variables.